ansible.cfg

```diff
@@ -4,7 +4,3 @@ inventory = inventory
 
 [ssh_connection]
 pipelining = True
-
-[privilege_escalation]
-become = True
```

Comment:
You can leave the `become = True`, and remove `--become` above. This one doesnt cause any prompt, so there is no disadvantage setting it.